public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -92,6 +95,79 @@ a#i18n-attribute
   The Angular extraction tool preserves both the _meaning_ and the _description_ in the translation source file
   to facilitiate contextually-specific translations.
 
+  As an alternative, if you want to translate some content without having to create a new element, you can use a `<ng-container>`
+  which doesn't get renderered in the DOM:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-ng-container', 'app/app.component.html')(format=".")
+
+:marked
+  You can also use a comment to get the same result:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-with-comment', 'app/app.component.html')(format=".")
+
+a#translating-attributes
+:marked
+  ## Translating attributes with _i18n-x_
+  Now you add an image to your html and since you are concerned with a11y, you add an `alt` tag to it:
+
++makeExample('cb-i18n/ts/app/app.component.1.html', 'i18n-alt', 'app/app.component.html')(format=".")
+
+:marked
+  Such attribute needs to be translated as well, and `Angular` provides you another attribute: `i18n-x` where you replace
+  the `x` with the name of the attribute to translate. So the previous example would end like:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-alt-translate', 'app/app.component.html')(format=".")
+
+:marked
+  This attribute also supports defining a meaning and description like `i18n`.
+
+a#plural
+:marked
+  ## Plurals
+  You have an application that counts animals. You need to keep in mind that it is `one wolf` but also `two wolves`.
+  Let's see how can you do that:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-plural', 'app/app.component.html')(format=".")
+
+:marked
+  * The first parameter is the key, which is a variable in your component that counts how many wolves we have.
+  * The second parameter indicates that you are using the `plural form`.
+  * The third parameter is the matches, in other words, how you define your plurals.
+
+  So how do you define your plurals? It uses [CLDR data](https://en.wikipedia.org/wiki/Common_Locale_Data_Repository).
+
+  CLDR uses some tags to define plural categories, here is a few of them:
+
+  * zero / =0
+  * one / =1
+  * five / =5
+  * nine / =9
+  * few
+  * other
+
+  You can use either a word like `zero` or `=0`.
+
+  Along the category, you add the default _English_ translation, so for 2 wolves you can say `two {two wolves}` or `=2 {two wolves}`.
+  If you use `other`, any non matched number will use the `other` translation.
```

Comment:
Note for later: `#` will be replaced by the count value as speced in ICU (not supported yet)

Explain that value match first (=...) then categories whatever the order in the src